doc/userguide/src/ExecutingTestCases/ConfiguringExecution.rst

```diff
@@ -130,6 +130,17 @@ combining individual tags or patterns together::
    --exclude xxORyyORzz
    --include fooNOTbar
 
+Starting from RF 5.0, it is also possible to use the reserved 
+tag `robot:exclude` to achieve
+the same effect as with using the `--exclude` option:
+
+.. sourcecode:: robotframework
+
+   *** Test Cases ***
+   Example
+      [Tags]    robot:exclude
+      Fail      this test will be excluded
```

Comment:
This message is pretty confusing considering the test is _not_ executed. Adding the word "not" to the message or changing the whole `Fail    this test ...` to `This is not executed` would be good.